index.html

```diff
@@ -676,6 +806,9 @@ <h3>
               <li>Initialize |gamepad|.{{Gamepad/[[buttons]]}} to the result of
               [=initializing buttons=] for |gamepad|.
               </li>
+              <li>Initialize |gamepad|.{{Gamepad/[[touches]]}} to the result of
+              [=initializing touches=] for |gamepad|.
```

Comment:
Instead, of putting the result at the end, let's just increment as we go but we need to make sure the this is locked thread safe.